packages/@okta/vuepress-site/docs/reference/api/groups/index.md

```diff
@@ -732,6 +742,131 @@ Link: <https://${yourOktaDomain}/api/v1/groups?after=00g1emaKYZTWRYYRRTSK&limit=
 ]
 ```
 
+#### List Groups with Search
+
+<ApiLifecycle access="ea" />
+
+Searches for groups based on the properties specified in the search parameter
+
+Property names in the search parameter are case sensitive, whereas operators (`eq`, `sw`, etc.) and string values are case insensitive.
+
+This operation:
+
+* Supports [pagination](/docs/reference/api-overview/#pagination).
+* Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding).
+For example, `search=type eq "OKTA_GROUP"` is encoded as `search=type+eq+%22OKTA_GROUP%22`.
+Examples use cURL-style escaping instead of URL encoding to make them easier to read.
+Use an ID lookup for records that you update to ensure your results contain the latest data.
+* Searches many properties:
+   - Any group profile property, including imported app group profile properties.
```

Comment:
@donwoods-okta With the group profiles project, do we plan on supporting searching:
- any custom Okta group profile property, or
- any custom app group profile property that is added in the future?
cc @nghadge-okta 

Just want to make sure we don't accidentally promise too much on this line if we're not sure yet.

Reply:
Certainly it is correct at this time: imported app group profile properties can be searched.  When we add support for custom properties in Okta groups, those properties will also be searchable.  We have not designed support for custom app group properties, but I expect those will also be searchable.